src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java

```diff
@@ -123,6 +137,10 @@ default ThrowableProblem toProblem(final Throwable throwable, final StatusType s
         problem.setStackTrace(stackTrace);
         return problem;
     }
+
+    default ThrowableProblem toProblem(final Throwable throwable, final HttpStatus status) {
+        return toProblem(throwable, new HttpStatusToStatusTypeConverter().convert(status));
```

Comment:
I was expecting to find a method like this

```java
default StatusType convert(final HttpStatus status) {
    // ...
}
```

Concrete advice implementations could then decide to override it. Having a separate concrete (non-public, final) class for it makes it harder to extend. Alternatively I could imagine having a separate interface with that single default method in there and `AdviceTrait` extending it.